include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -5144,10 +5073,6 @@ namespace internal
                                   Number::size()> &  constraint_mask,
                  Number *                            values)
     {
-      const Number *weights = fe_eval.get_shape_info()
-                                .data.front()
-                                .subface_interpolation_matrix.data();
-
```

Comment:
Can we, instead of writing the full specification further down, simply write:
```c++
const auto interpolation_matrices = fe_eval.get_shape_info()
                          .data.front()
                          .subface_interpolation_matrices;
```
once here, and then further down simply say
```c++
  const Number *weights = interpolation_matrices[is_subface_0].data();
```